templates/layout.html

```diff
@@ -55,8 +57,8 @@
 </head>
 
 <body>
+    {{ logo_def() }}
     <ul class='navbar center'>
-
         <li class='col3 openstreetmap-us'>
             <a href='/' class="button active">
               <svg class="iconsvg pre-text"><use xlink:href="#osm-logo"></use></svg>
```

Comment:
A little background ..  
This line `<use>`'s the logo defined elsewhere as a `<symbol>`.  
In most browsers, you could say `<use xlink:href="spritesheet.svg#whatever">` but IE is not smart enough to fetch the spritesheet.  It can only `<use>` something that already exists elsewhere in the document.